### PR TITLE
Fixes #7306: Fix rudder-dev bugs when using alt redmine

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/7306

### CODE REVIEW

**Comment by peckpeck** [comment]
PR updated

**Comment by peckpeck** [comment]
PR updated

File: scripts/rudder-dev/rudder-dev [hunk 1]
```diff
@@ -857,6 +878,10 @@ def unstash():
   # get current branch (the global variable may not be accurate in flight)
   branch_process = Popen("git rev-parse --abbrev-ref HEAD", shell=True, stdout=PIPE)
   this_branch = branch_process.stdout.read().strip()
+  pprint(this_branch)
+  pprint(current_branch)
+  pprint(UPSTREAM_REPOSITORY)
+  pprint("git ls-remote -h " + UPSTREAM_REPOSITORY + " " + this_branch + " | wc -l")
```
Comment (VinceMacBuche) [inline scripts/rudder-dev/rudder-dev]: Not sure those pprints should be here :)

File: scripts/rudder-dev/rudder-dev [hunk 2]
```diff
@@ -1189,6 +1215,7 @@ def takeover(ticket):
   commit_push(current_branch, "Work in progress", True)
 
   # Set ticket's status
+  info = get_ticket_info(ticket_id, internal)
   ticket_to_in_progress(ticket_id, "I'm taking over this ticket!")
```
Comment (VinceMacBuche) [inline scripts/rudder-dev/rudder-dev]: This should change to 
  ticket_to_in_progress(info, "I'm taking over this ticket!")

**Comment by peckpeck** [comment]
PR updated

**Pull request merged**